docs/guides/redis/monitoring/using-builtin-prometheus.md

```diff
@@ -58,318 +61,258 @@ spec:
+
+**Create ConfigMap:**
+
+At first, create a ConfigMap with the scrapping configuration. Bellow, the YAML of ConfigMap that we are going to create in this tutorial.
 
 ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: prometheus-server-conf
+  name: prometheus-config
   labels:
-    name: prometheus-server-conf
-  namespace: demo
+    app: prometheus-demo
+  namespace: monitoring
 data:
   prometheus.yml: |-
     global:
       scrape_interval: 5s
       evaluation_interval: 5s
     scrape_configs:
     - job_name: 'kubernetes-service-endpoints'
-
       kubernetes_sd_configs:
       - role: endpoints
-
+      # by default Prometheus server select all kubernetes services as possible target.
+      # relabel_config is used to filter only desired endpoints
       relabel_configs:
+      # keep only those services that has "prometheus.io/scrape: true" anootation.
       - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_scrape]
         action: keep
         regex: true
+      # read the scrapping scheme from "prometheus.io/scheme: <scheme>" annotation
       - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_scheme]
         action: replace
         target_label: __scheme__
         regex: (https?)
+      # read the metric path from "prometheus.io/path: <path>" annotation
       - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_path]
         action: replace
         target_label: __metrics_path__
         regex: (.+)
+      # read the port from "prometheus.io/port: <port>" annotation and update scrapping address accordingly
       - source_labels: [__address__, __meta_kubernetes_service_annotation_prometheus_io_port]
         action: replace
         target_label: __address__
         regex: ([^:]+)(?::\d+)?;(\d+)
         replacement: $1:$2
       - action: labelmap
         regex: __meta_kubernetes_service_label_(.+)
+      # add service namespace as label to the scrapped metrics
       - source_labels: [__meta_kubernetes_namespace]
         action: replace
         target_label: kubernetes_namespace
+      # add service name as label to the scrapped metrics
       - source_labels: [__meta_kubernetes_service_name]
         action: replace
         target_label: kubernetes_name
 ```
 
-Create above ConfigMap
+Let's create above `ConfigMap`,
 
 ```console
-$ kubectl create -f https://raw.githubusercontent.com/kubedb/cli/0.9.0/docs/examples/monitoring/builtin-prometheus/demo-1.yaml
-configmap/prometheus-server-conf created
+$ kubectl apply -f https://raw.githubusercontent.com/kubedb/cli/0.9.0/docs/examples/monitoring/builtin-prometheus/prom-config.yaml
+configmap/prometheus-config created
 ```
 
-Now, the below yaml is used to deploy Prometheus in kubernetes:
+**Create RBAC:**
```

Comment:
Why do we add this section separately? We should just add this into the deployment.yaml and always create the RABC objects.

Reply:
Just user to make aware of what RBAC permissions are given.